webui/module/Client/src/Client/Controller/ClientController.php

```diff
@@ -328,161 +328,29 @@ public function getDataAction()
         $this->bsock = $this->getServiceLocator()->get('director');
         $clients = $this->getClientModel()->getClients($this->bsock);
         $dot_clients = $this->getClientModel()->getDotClients($this->bsock);
-        $dird_version = $this->getDirectorModel()->getDirectorVersion($this->bsock);
         $this->bsock->disconnect();
       }
       catch(Exception $e) {
         echo $e->getMessage();
       }
 
-      $bareos_updates = json_decode($_SESSION['bareos']['product-updates'], true);
-
-      if(array_key_exists('obsdistribution', $dird_version)) {
-        $dird_dist = $dird_version['obsdistribution'];
-      }
-      else {
-        $dird_dist = null;
-      }
-
-      if(array_key_exists('obsarch', $dird_version)) {
-        if(preg_match("/debian/i", $dird_dist) && $dird_version['obsarch'] == "x86_64") {
-          $dird_arch = "amd64";
-        }
-        elseif(preg_match("/ubuntu/i", $dird_dist) && $dird_version['obsarch'] == "x86_64") {
-          $dird_arch = "amd64";
-        }
-        elseif(preg_match("/windows/i", $dird_dist) && $dird_version['obsarch'] == "Win32") {
-          $dird_arch = "32";
-        }
-        elseif(preg_match("/windows/i", $dird_dist) && $dird_version['obsarch'] == "Win64") {
-          $dird_arch = "64";
-        }
-        else {
-          $dird_arch = $dird_version['obsarch'];
-        }
-      }
-      else {
-        $dird_arch = null;
-      }
-
-      if(array_key_exists('version', $dird_version)) {
-        $dird_vers = $dird_version['version'];
-      }
-      else {
-        $dird_vers = null;
-      }
-
       $result = array();
+
       for($i = 0; $i < count($clients); $i++) {
         $result[$i]['clientid'] = $clients[$i]['clientid'];
         $result[$i]['uname'] = $clients[$i]['uname'];
         $result[$i]['name'] = $clients[$i]['name'];
         $result[$i]['autoprune'] = $clients[$i]['autoprune'];
         $result[$i]['fileretention'] = $clients[$i]['fileretention'];
         $result[$i]['jobretention'] = $clients[$i]['jobretention'];
-        $result[$i]['installed_fd'] = "";
-        $result[$i]['available_fd'] = "";
-        $result[$i]['installed_dird'] = "";
-        $result[$i]['available_dird'] = "";
-        $result[$i]['enabled'] = "";
-
+        $uname = explode(",", $clients[$i]['uname']);
+        $v = explode(" ", $uname[0]);
+        $result[$i]['version'] = $v[0];
```

Comment:
```suggestion
        $result[$i]['version'] = $v[0];
        $result[$i]['enabled'] = "";
```
As `$result[$i]['enabled']` is still used, it can't hurt to set a default value.